src/apps/AppCenter/DiscoverApps/icons/dandelion.svg

```diff
@@ -0,0 +1 @@
38a4.96 4.96 0 010-1.875h-4.533a2.814 2.814 0 010 1.875h4.533z" fill="#DDE4E9"/><path d="M40.085 31.047a.937.937 0 00-1.149-.663l-2.743.735a.934.934 0 00-.663 1.149l.735 2.743a.938.938 0 001.811-.485l-.149-.558.649.375a.937.937 0 10.937-1.624l-.648-.374.557-.15a.937.937 0 00.663-1.148z" fill="#fff"/><path d="M34 30.619c.162-.281.367-.527.607-.732L30.68 27.62a4.956 4.956 0 01-.937 1.624l3.925 2.266c.058-.31.169-.61.33-.891z" fill="#DDE4E9"/><path d="M11.915 20.953c.134.5.648.797 1.149.663l2.743-.735a.934.934 0 00.663-1.149l-.735-2.743a.937.937 0 10-1.811.485l.149.558-.649-.375a.938.938 0 10-.937 1.624l.649.374-.558.15a.938.938 0 00-.663 1.148z" fill="#fff"/><path d="M21.71 23.523a4.95 4.95 0 01.546-.767l-3.925-2.266a2.815 2.815 0 01-.938 1.623l3.926 2.267c.101-.292.23-.58.39-.857z" fill="#DDE4E9"/><path d="M35.011 36.265l-2.743-.735a.934.934 0 00-1.148.663l-.735 2.743a.937.937 0 101.81.486l.15-.558.375.65a.938.938 0 001.623-.938l-.374-.65.557.15a.937.937 0 00.485-1.811z" fill="#fff"/><path d="M31.51 33.67l-2.266-3.926a4.951 4.951 0 01-1.624.937l2.267 3.926a2.813 2.813 0 011.624-.938z" fill="#DDE4E9"/><path d="M16.989 15.735l2.743.735a.934.934 0 001.148-.663l.736-2.743a.938.938 0 00-1.812-.486l-.149.558-.375-.65a.937.937 0 10-1.623.938l.374.65-.557-.15a.938.938 0 00-.485 1.811z" fill="#fff"/><path d="M20.49 18.33l2.266 3.926a4.96 4.96 0 011.624-.937l-2.267-3.926a2.813 2.813 0 01-1.624.938z" fill="#DDE4E9"/><path d="M20.88 36.193a.935.935 0 00-1.148-.663l-2.743.735a.937.937 0 10.485 1.811l.558-.149-.375.649a.937.937 0 101.624.937l.374-.648.15.557a.938.938 0 001.81-.485l-.734-2.744z" fill="#fff"/><path d="M23.523 30.29a4.95 4.95 0 01-.767-.546l-2.266 3.925a2.815 2.815 0 011.623.938l2.267-3.926a4.957 4.957 0 01-.857-.39z" fill="#DDE4E9"/><path d="M31.12 15.807a.935.935 0 001.148.663l2.743-.735a.937.937 0 10-.485-1.811l-.558.149.375-.649a.938.938 0 00-1.624-.937l-.374.649-.15-.558a.938.938 0 00-1.81.486l.734 2.743z" fill="#fff"/><path d="M28.477 21.71c.278.16.533.344.767.546l2.266-3.925a2.813 2.813 0 01-1.623-.938L27.62 21.32c.292.101.58.23.857.39z" fill="#DDE4E9"/><path d="M16.376 31.556a.934.934 0 00-.569-.437l-2.743-.735a.937.937 0 10-.486 1.812l.558.149-.65.374a.938.938 0 00.938 1.624l.65-.374-.15.557a.937.937 0 101.811.485l.735-2.743a.934.934 0 00-.094-.712z" fill="#fff"/><path d="M18 30.619c.162.28.273.582.33.892l3.926-2.267a4.96 4.96 0 01-.937-1.624l-3.926 2.267c.24.205.445.45.607.732z" fill="#DDE4E9"/><path d="M35.624 20.444c.12.207.319.37.569.437l2.743.735a.937.937 0 10.486-1.812l-.558-.149.65-.375a.938.938 0 00-.938-1.623l-.65.374.15-.557a.938.938 0 00-1.81-.485l-.736 2.743a.934.934 0 00.094.712z" fill="#fff"/><path d="M34.607 22.113a2.811 2.811 0 01-.938-1.624l-3.925 2.267a4.96 4.96 0 01.937 1.624l3.926-2.267z" fill="#DDE4E9"/><defs><linearGradient id="paint0_linear" x1="-20.307" y1="-25.15" x2="54.032" y2="55.511" gradientUnits="userSpaceOnUse"><stop stop-color="#7C80F2"/><stop offset="1" stop-color="#7C80F2"/></linearGradient></defs></svg>
```

Comment:
Just wondering if we could make this icon square like the others?

Reply:
Ah hmm, it doesn't look like there's a noticeable difference here.

I mentioned I've had some stashed changes for this a while back, but I'd like to un-duplicate these "known app" assets since we have them in the app center, templates, and I think one or two other places 😄.

Reply:
> Ah hmm, it doesn't look like there's a noticeable difference here.

No there shouldn’t be, it’s really just in case we change the applied radius later.

> I mentioned I've had some stashed changes for this a while back, but I'd like to un-duplicate these "known app" assets since we have them in the app center, templates, and I think one or two other places :smile:.

Cool yes, let’s do this at this point. We also have different sizes depending on the apps, so we’ll align everything then.